pkg/daemon/ceph/osd/agent.go

```diff
@@ -166,10 +171,19 @@ func (a *OsdAgent) configureDevices(context *clusterd.Context, devices *DeviceOs
 	}
 
 	var osds []oposd.OSDInfo
+	var lvPath string
+	var skipLVRelease bool
 	if devices == nil || len(devices.Entries) == 0 {
 		logger.Infof("no more devices to configure")
 		if cvSupported {
-			return getCephVolumeOSDs(context, a.cluster.Name, a.cluster.FSID, "")
+			if a.pvcBacked {
+				lvPath, err = getDeviceLVPath(context, fmt.Sprintf("/mnt/%s", a.nodeName))
+				if err != nil {
+					return osds, fmt.Errorf("failed to get logical volume path. %+v", err)
+				}
+				skipLVRelease = true
```

Comment:
Why do we need to skip the LV release? Because the OSD daemon is already running and active on the LV?